pkgs/development/ocaml-modules/fmt/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchurl, ocaml, findlib, ocamlbuild, opam, topkg, cmdliner }:
+
+stdenv.mkDerivation {
+  name = "ocaml${ocaml.version}-fmt-0.8.0";
+
+  src = fetchurl {
+    url = http://erratique.ch/software/fmt/releases/fmt-0.8.0.tbz;
+    sha256 = "16y7ibndnairb53j8a6qgipyqwjxncn4pl9jiw5bxjfjm59108px";
+  };
+
+  unpackCmd = "tar xjf $src";
+
+  buildInputs = [ ocaml findlib ocamlbuild opam topkg cmdliner ];
+
+  inherit (topkg) buildPhase;
+
+  createFindlibDestdir = true;
+
+  installPhase = ''
```

Comment:
That is a good point, `opam-installer` does not necessarily need it, and `buildOcaml` should know the library name as the `name` attribute is the name without version and ocaml-prefix.